app/assets/stylesheets/govuk_publishing_components/components/_error-summary.scss

```diff
@@ -2,6 +2,6 @@
 @import "govuk/components/error-summary/error-summary";
 
 .gem-c-error-summary__list-item {
-  @include govuk-font($size: 19, $weight: bold);
   color: $govuk-error-colour;
+  @include govuk-font($size: 19, $weight: bold);
```

Comment:
Just to check - this doesn't override the colour set on the line above? I think the mixin might set color to inherit?

Reply:
No it doesn't. This is the resulting rule:

```
.gem-c-error-summary__list-item {
    color: #d4351c;
    font-family: "GDS Transport",arial,sans-serif;
    -webkit-font-smoothing: antialiased;
    -moz-osx-font-smoothing: grayscale;
    font-weight: 700;
    font-size: 1rem;
    line-height: 1.25
}
```